README.md

```diff
@@ -1 +1,58 @@
-# JS-Capstone-Project-
\ No newline at end of file
+# Javascript Capstone Project
+> An API based web app that displays Meals from the MealsDB API and the Involvement API from Microverse. The features of this webapp include
+
+An option to like a meal by clicking on the like button
+A comment button that opens up a modal and allows users to leave comments.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript(ES6)
+- Webpack
+- Jest Library
+
+## Live Demo
+- at http://localhost:8080/
+
+## Getting Started
+
+### To get a local copy run the following steps:
+
+- [Copy this link](https://github.com/mbdesigns1989/To-do-List/tree/To-Do-list-add-and-remove)
```

Comment:
- [ ] There is a typo with this link you used link for To-DO-list, kindly attach the right link instead of this one.

Reply:
Well solved